bin/autodeploy.sh

```diff
@@ -0,0 +1,416 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2087
+set -Eeuo pipefail
+
+msg() {
+  echo >&2 -e "${1-}"
+}
+
+trap cleanup SIGINT SIGTERM ERR EXIT
+
+usage() {
+  cat <<EOF
+Usage: $(basename "${BASH_SOURCE[0]}") [-h] [-v] [--force-redeploy] [--artifact-hash string] [--target-system string]
+
+This script deploys "Wire in a box", a functional, self-contained demo environment of Wire and it's various components.
+Following this Readme: https://github.com/wireapp/wire-server-deploy/blob/master/offline/docs_ubuntu_22.04.md
+
+This script can either be run locally or via CI, always within the context of it's wire-server-deploy repository.
+
+Requirements:
+ * A dedicated (root) server running Ubuntu 22.04 Server with at least 16 CPU cores, 64 GiB RAM, 500 GB NVMe SSD and an (unfirewalled) public IP address.
+ * Access to server via root user using an SSH keypair.
+
+Deployment takes about 90 to 100 minutes on a Hetzner "AX41-NVMe" target system. Longer if RAID is actively syncing disks.
+
+Running the script without any arguments requires one interaction - confirming the removal of any resources left by a previous Wire-in-a-box installation.
+For CI usage, it's recommended to invoke "--force-redeploy".
+
+It is likely desirable to invoke the script with "--artifact-hash" and / or "--target-system" as well. These are the hardcoded fallback values:
+ * artifact-hash = 5c06158547bc57846eadaa2be5c813ec43be9b59
+ * target-system = wiab-autodeploy.wire.link
+
+Available options:
+-h, --help          Print this help and exit
+-v, --verbose       Print script debug info
+--force-redeploy    Force cleanup of previous Wire-in-a-box installation on target host
+--artifact-hash     String, specifies artifact ID as created here: https://github.com/wireapp/wire-server-deploy/actions/workflows/offline.yml
+                    Defaults to 5c06158547bc57846eadaa2be5c813ec43be9b59
+--target-system     String, domain name used to access the target host
```

Comment:
I think it would be good if this parameter contained the "domain" string somewhere, I initially expected it was like the OS version.

Reply:
Good point, I've renamed the `--target-system` flag to `--target-domain`.